proto/stream_plan.proto

```diff
@@ -1041,13 +1041,14 @@ message StreamFragmentGraph {
   // If none, default parallelism will be applied.
   Parallelism parallelism = 6;
 
-  // Expected vnode count for the graph.
+  // Specified max parallelism, i.e., expected vnode count for the graph.
+  //
   // The scheduler on the meta service will use this as a hint to decide the vnode count
   // for each fragment.
   //
   // Note that the actual vnode count may be different from this value.
   // For example, a no-shuffle exchange between current fragment graph and an existing
   // upstream fragment graph requires two fragments to be in the same distribution,
   // thus the same vnode count.
-  uint32 expected_vnode_count = 7;
+  uint32 max_parallelism = 7;
```

Comment:
Personally, I'd like to use `vnode_count` for all internal occurrences, and only use "max_parallelism" for the user-facing part. Because we (RW developers) are familiar with vnode, so `vnode_count` sounds like the most straight-forward naming.

Reply:
My major concern is that, `vnode_count` of a streaming job may not be a physical concept at all. But it's true that prefixing with `expected` makes it doesn't sounds that confusing. Will change.

Reply:
Upon further consideration, I believe `max_parallelism` is more understandable because it aligns better with existing job attributes like `parallelism` (or `assigned_parallelism`). When the scope goes to fragment, it'll still be named as `vnode_count`.